config/routes.rb

```diff
@@ -58,6 +58,9 @@
       post '/pointrequest', to: 'evaluations#submit_point_request'
       delete '/pointrequest', to: 'evaluations#cancel_point_request', as: :cancel_point_request
     end
+    get '/messages', to: 'messages#index'
+    get '/messages/all', to: 'messages#all'
+    post '/messages', to: 'messages#create', as: :message_create
```

Comment:
Why don't we use sth like the delegate below?
I mean `post '/message', to: 'messages#create'`
Nevertheless, these does not really look like the REST/CRUD model, but currently i don't have a better idea.